CHANGELOG.md

```diff
@@ -1,3 +1,12 @@
+# v93.0.1 (_2022-04-20_)
+
+[Full Changelog](https://github.com/mozilla/application-services/compare/v93.0.0...v93.0.1)
+
+## Nimbus FML ⛅️🔬🔭🔧
+
+### What's Fixed
+  - Handling of optional types which require a mapping to a usable type. ([#4915](https://github.com/mozilla/application-services/pull/4915))
+
```

Comment:
What should we do about the changelog entry from `v91.1.2`?

```
## Places
- Downgraded places `get_registered_sync_engine` `log:error` to `log:warn` to fix an issue where places was unnecessarily creating sentry noise
```

I think it's slightly confusing now, because users might assume that that fix was in `v93.0.0` as well, which is incorrect.  I'm not sure what the best way to do this, but I think I favor removing the `v91.1.2` section and moving that note to the `v93.0.1` section.   That way the changelog reflects the fact that `v91.1.2` was from a branch that diverged from `main`.

Reply:
I would opt to keep it under v91.1.2 and duplicate it under v93.0.1, and note down explicitly in both places that v91.1.2 was a release not off the main branch

Just in the unlikely case that someone is looking through our changelog in `main` for information about v91.1.2 🤷